### ISSUE TITLE
I already have thinkfan installed?

### ISSUE THREAD

**Issue by kendoori** [zocker-160/thinkfan-ui#2]
I have an existing thinkfan.conf that's in /etc/

What is the impact of my adding thinkfan-ui on top of that, or should I remove thinkfan set up I have.

I am running Linut Mint 20.2 on a Thinkpad T480.

**Comment by zocker-160** [comment]
This application does not conflict with the already existing `thinkfan` software, since for now it only is able to directly send manual fan commands to the ThinkPad fan interface.

In an upcoming version I will be adding an option to tweak the fan curve config file of `thinkfan`, so both applications will be working hand in hand.

There is no need to remove `thinkfan`, since this application does not replace it, actually it will be required to be installed in upcoming versions of `thinkfan-ui`.

Hope this helps ;)

**Comment by alexandru-tigaeru** [comment]
Hi,
do you still plan to add the option to tweak the fan curve?
Cheers

**Comment by zocker-160** [comment]
yes I do, but since I have been working on other projects, this has not see any progress so far.

**Comment by depado** [comment]
Subscribing to this issue because I'm also facing this conflict, if I can be of any help (testing or something else) don't hesitate

**Comment by zocker-160** [comment]
@Mentalist @depado the issue should now be fixed in AUR.

**Comment by depado** [comment]
Works like a charm, thank you!
